socs/agents/smurf_file_emulator/agent.py

```diff
@@ -475,7 +505,8 @@ def uxm_setup(self, session, params):
         """
         self._run_setup(action='uxm_setup', action_time=time.time(),
                         session=session)
-        time.sleep(1)
+        if params.get('sleep', True):
```

Comment:
Create a `Parameters:` section in the docstring with the new 'sleep' parameter and the old 'test_mode' parameter.

(`@ocs_agent.param` decorators could also be used.)

Reply:
Not sure where test_mode was copied from, but it's not actually implemented here... so I just removed from the spec line.